src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -128,9 +133,11 @@ def parse_body(self, data, decode=False):
             try:
                 res = json.loads(data)
             except ValueError as exc:
-                msg = 'Unable to load JSON data\n%s' % data
-                raise ValueError(exc)
-            return res
+                msg = 'Unable to load JSON data, err=%s, data type=%s' \
+                        % (str(exc), type(data))
+                print msg
```

Comment:
Ah, I didn't know about that. Thanks for explaining. I was just wondering when we move to python 2.7, didn't we say that we will remove the print statement, The problem with 'print' in WMAgent, it doesn't log to ComponentLog.